connect/mirror-client/src/test/java/org/apache/kafka/connect/mirror/MirrorClientTest.java

```diff
@@ -159,4 +191,12 @@ public void remoteTopicsSeparatorTest() throws InterruptedException {
         assertTrue(remoteTopics.contains("source3__source4__source5__topic6"));
     }
 
+    public void testIdentityReplicationTopicSource() {
+        MirrorClient client = new FakeMirrorClient(
+            new IdentityReplicationPolicy("primary"), Arrays.asList());
+        assertEquals("topic1", client.replicationPolicy()
+            .formatRemoteTopic("primary", "topic1"));
```

Comment:
Should we also try `formatRemoteTopic()` with a heartbeat topic?

Reply:
good call, thanks! fixed.